src/Components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  error: string | null;
+  setError: React.Dispatch<React.SetStateAction<string | null>>,
```

Comment:
The setError prop is very generic and can be a little difficult to understand. Consider renaming this to something more descriptive, like 'clearError'.